src/main/java/views/CureReportViewController.java

```diff
@@ -131,14 +131,15 @@ private void initializeEmployeeListCb() {
 
     employeeObsList =
         FXCollections.observableArrayList(UserController.getInstance().getAllNurses());
+    employeeObsList.add(0, null);
     employeeObsList.addAll(UserController.getInstance().getAllAdministrators());
     selectEmployeeCb.setItems(employeeObsList);
 
-    selectEmployeeCb.setCellFactory(this::displayUser);
-    selectEmployeeCb.setButtonCell(selectEmployeeCb.getCellFactory().call(null));
+    selectEmployeeCb.setCellFactory(this::displayUserCell);
```

Comment:
se for 'empregado selecionado' o certo é selectedEmployeeCb, mas... mee irrelevante kkk

Reply:
n entendi .-.

Reply:
@HyanBatista nesse caso acho que o que queria ser dito é a "ComboBox para selecionar de empregados", mas entendi o que tu quis dizer. Pra mim `selectedEmployeeCb` não faz tanto sentido pq vira a "combo box do empregado selecionado" e não o "empregado selecionado" tkgd